substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -1997,6 +2000,13 @@ pub mod env {
 		self.terminate(memory, beneficiary_ptr)
 	}
 
+	/// Returns the amount of ref_time left.
+	/// See [`pallet_revive_uapi::HostFn::ref_time_left`].
+	fn ref_time_left(&mut self, memory: &mut M) -> Result<u64, TrapReason> {
+		self.charge_gas(RuntimeCosts::RefTimeLeft)?;
+		Ok(self.ext.gas_meter().gas_left().ref_time())
+	}
```

Comment:
Shouldn't this be marked stable and then sorted with the stable functions?

Reply:
Ah yes of course, changed it thanks